src/app/routes/liveRadio/getInitialData/index.js

```diff
@@ -19,10 +22,55 @@ const getPageIdentifier = path([
 const getHeading = path(['content', 'blocks', 0, 'text']);
 const getBodySummary = path(['content', 'blocks', 1, 'text']);
 
-export default async ({ path: pathname }) => {
+export const hasRadioSchedule = async (service, pathname) => {
+  const config = await getConfig(service);
+
+  if (service === 'persian' && pathname.includes('bbc_persian_radio')) {
```

Comment:
Same issue here, trying to change logic to avoid this hardcoded Persian value

Reply:
Leaving this until toggles are on iSite next week - https://github.com/orgs/bbc/projects/53#card-42064710